test/w3c/w3ctest.js

```diff
@@ -56,31 +55,76 @@ function createJsdom(urlPrefix, testPath, t) {
   });
 }
 
+var childProcess = require("child_process");
+var EventEmitter = require("events");
+var dns = require("dns");
+
 module.exports = function (exports) {
-  const staticFileServer = st({ path: path.resolve(__dirname, "tests"), url: "/", passthrough: true });
-  const server = http.createServer(function (req, res) {
-    staticFileServer(req, res, function () {
-      fallbackToHostedVersion(req, res);
+  var server = new EventEmitter();
+  server.started = false;
+
+  dns.lookup("web-platform.test", function (err) {
```

Comment:
No because w3c/web-platform-tests server needs them and does not just serve static files, there is specific paths for testing cross domains, headers, uploads, redirects, etc.